code-of-conduct.md

```diff
@@ -0,0 +1,68 @@
+# Code of Conduct
+
+Helix Launcher bases it's policies around a Code of Conduct, it
+provides behavoural expectations for all members interacting in our
+community. **Not reading the Code of Conduct is not an excuse for
+breaking it**
+
+## The Pledge
+
+We as community members, contributors, and team members pledge to
+make our community a harassment-free experience for everyone, no
+matter one's age, culture, ethnicity, gender identity or expression,
+nationality, physical or mental disabilities, neurodivergencies,
+politics, race, religion, sex, sexual orientation, socio-economic
+status or subcultural affiliation.
+
+We pledge to act and interact in ways that contribute to an open,
+welcoming, diverse, inclusive, and healthy community, and we are
+committed to promoting these concepts.
+
+## Standards
+
+We expect community members to act in a positive, authentic, and
+deliberate way. Consideration and respect provides a basis for a
+solid community, and we request our members actions reflect these
+values. We also expect communities to refrain from talking about, or
+encouraging illegal activities, including, but not limited to,
```

Comment:
I maintain that including "illegal activities" in a Code of Conduct dilutes its value. This should be in a separate document for rules of official community spaces.
Either way, a jurisdiction should be mentioned. Complying with the laws of *every* jurisdiction would necessitate directly breaking other parts of the CoC.